install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```
@@ -73,11 +73,11 @@ spec:
       rules:
         - alert: MCDPivotError
           expr: |
-            mcd_pivot_err > 0
+            increase(mcd_pivot_errors_total[15m]) > 0)
           labels:
             severity: warning
           annotations:
-            message: "Error detected in pivot logs on {{ $labels.node }} "
+            message: "Error detected in pivot logs on {{ $labels.node }} , upgrade may be blocked. For more details:  oc logs -f -n openshift-machine-config-operator {{ $labels.pod }} -c machine-config-daemon "
```

This is nice! Probably worth doing the same for the other alert?

```suggestion
            message: "Error detected in pivot logs on {{ $labels.node }} , upgrade may be blocked. For more details:  oc logs -f -n {{ $labels.namespace }} {{ $labels.pod }} -c machine-config-daemon "
```

---

great suggestion, fixed.